content/docs/userspace/gall-guide/7-sur-and-marks.md

```diff
@@ -0,0 +1,375 @@
++++
+title = "7. Structures and Marks"
+weight = 35
+template = "doc.html"
++++
+
```

Comment:
I think it would be helpful to include a statement saying that the code presented in this section will not yet build a fully-functional Gall agent, but that it will be completed in the next section.